webpack.config.js

```diff
@@ -12,24 +12,29 @@ const webpack = require( 'webpack' );
 const { bundler, styles } = require( '@ckeditor/ckeditor5-dev-utils' );
 const CKEditorWebpackPlugin = require( '@ckeditor/ckeditor5-dev-webpack-plugin' );
 const BabiliPlugin = require( 'babel-minify-webpack-plugin' );
-const buildConfig = require( './build-config' );
 
 module.exports = {
 	devtool: 'source-map',
 
 	entry: path.resolve( __dirname, 'src', 'ckeditor.js' ),
 
 	output: {
+		// The name under which the editor will be exported.
+		library: 'ClassicEditor',
+
 		path: path.resolve( __dirname, 'build' ),
 		filename: 'ckeditor.js',
 		libraryTarget: 'umd',
-		libraryExport: 'default',
-		library: buildConfig.moduleName
+		libraryExport: 'default'
 	},
 
 	plugins: [
 		new CKEditorWebpackPlugin( {
-			language: buildConfig.config.language,
+			// UI language. Language codes follow the https://en.wikipedia.org/wiki/ISO_639-1 format.
+			// When changing the builtin language, remember to also change it in the ditor's config (src/ckeditor.js).
```

Comment:
// When changing the built-in language, remember to also change it in the editor's configuration (src/ckeditor.js).